scripts/normalize-by-median.py

```diff
@@ -344,6 +344,11 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
         else:
             output_name = args.single_output_file.name
         outfp = args.single_output_file
```

Comment:
The logic around single output which may-or-may not be STDOUT vs one output per input with '.keep' appended is a mess.

Since single_output_file is of `type=argparse.FileType('w')` then we can't re-open it (or cause it to be re-opened). So no setting `output_name = '/dev/stdout'`

What really should happen is that `if args.single_output_file:` then `outfp = args.single_output_file` and `output_name = args.single_output_file.name` end of story.